src/screens/import-account/import-wallet/import-wallet-from-keystore-file/import-wallet-from-keystore-file.tsx

```diff
@@ -68,7 +86,14 @@ export const ImportWalletFromKeystoreFile: FC<ImportWalletProps> = ({ onSubmit }
               <Divider size={formatSize(12)} />
               <Label label="File password" description="Please enter a password for keystore file" />
               <FormPasswordInput name="password" />
-              <FormCheckbox name="shouldUseFilePasswordForExtension">
+              <FormCheckbox
+                {...(values.shouldUseFilePasswordForExtension === true &&
```

Comment:
make `shouldUseFilePasswordForExtension` required and set default value `false` to remove these `true` checks